app/src/main/res/values/strings.xml

```diff
@@ -367,4 +367,5 @@
     <string name="add_patch_bundle">Add patch bundle</string>
     <string name="bundle_url">Bundle URL</string>
     <string name="auto_update">Auto update</string>
+    <string name="reset_api_url">Reset API URL</string>
```

Comment:
Move this to the other api url dialog strings and rename the resource accordingly